<!DOCTYPE html>
<html>
	<head>
		    <meta charset="utf-8">
		<title>CVE-2019-10904 - Vulnerability Data</title>
		    <link rel="shortcut icon" type="image/png" href="/favicon.ico" />
	</head>
<body>
<h2>Vulnerability Data</h2>
<h1>CVE-2019-10904</h1>
<p>Roundup 1.6 allows XSS via the URI because frontends/roundup.cgi and roundup/cgi/wsgi_handler.py mishandle 404 errors.</p>

<dl>
	<dt>Published</dt>
	<dd>2019-04-06</dd>

	<dt>Modified</dt>
	<dd>2019-04-07</dd>
</dl>

<h3>References</h3>
<ul>
	<li><a href="https://bugs.python.org/issue36391">MISC:https://bugs.python.org/issue36391</a></li>
<li><a href="https://github.com/python/bugs.python.org/issues/34">MISC:https://github.com/python/bugs.python.org/issues/34</a></li>
<li><a href="https://www.openwall.com/lists/oss-security/2019/04/05/1">MISC:https://www.openwall.com/lists/oss-security/2019/04/05/1</a></li>
<li><a href="https://lists.debian.org/debian-lts-announce/2019/04/msg00009.html">MLIST:[debian-lts-announce] 20190407 [SECURITY] [DLA 1750-1] roundup security update</a></li>
<li><a href="http://www.openwall.com/lists/oss-security/2019/04/07/1">MLIST:[oss-security] 20190407 Re: XSS in roundup bug tracker 404 page</a></li>
</ul>

<footer>
	<p><a href="https://reconmap.org">Track security vulnerabilities</a> with <strong>Reconmap</strong>.</p>
</footer>
